specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2022-11-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,1971 @@
+          "description": "SAS URI for the target directory to export to",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise export operation."
+    },
+    "OperationStatus": {
+      "description": "The status of a long-running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation's unique id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation's name."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the operation."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the operation."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse",
+          "description": "Error response describing why the operation failed."
+        }
+      }
+    },
+    "RegionSkuDetails": {
+      "title": "Region Sku Details",
+      "description": "List of details about all the available SKUs",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegionSkuDetail"
+          },
+          "description": "List of Sku Detail",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "LocationInfo": {
+      "type": "object",
+      "description": "Information about location (for example: features that it supports)",
+      "readOnly": true,
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location name"
+        },
+        "capabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          },
+          "description": "List of capabilities",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "SkuDetail": {
+      "type": "object",
+      "title": "Sku Detail",
+      "description": "Information about Sku",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The type of RedisEnterprise cluster to deploy. Possible values: (Enterprise_E10, EnterpriseFlash_F300 etc.)",
+          "enum": [
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500"
+          ],
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        }
+      },
+      "readOnly": true
```

Comment:
>"readOnly": true [](http://example.com/codeflow?start=6&length=16)

I'm not sure readonly at this level is meaningful, I thought it only works on properties.